aim/web/ui/src/pages/RunDetail/RunOverviewTab/GitInfoCard/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { IGitInfoCardProps } from '../RunOverViewTab.d';
+
+import './GitInfoCard.scss';
+
+function GitInfoCard(props: IGitInfoCardProps) {
+  return (
+    <Card title='Git Info Card'>
+      {_.isEmpty(props.data) ? (
+        <IllustrationBlock size='medium' title='No Results' />
+      ) : (
+        <div className='InfoSection ScrollBar__hidden flex fjb'>
+          <div className='InfoCard flex fdc'>
+            <Text
+              className='InfoCardLabel'
+              weight={400}
+              tint={50}
+              size={12}
+              color='primary'
+            >
+              Branch
+            </Text>
+            <div className='InfoCardValue flex fac'>
+              <Icon name='branch' fontSize={14} />
+              <Text
+                className='InfoCardValueText'
+                weight={500}
+                size={12}
+                color='primary'
+              >
+                {props.data?.branch}
+              </Text>
+            </div>
+          </div>
+          <div className='InfoCard flex fdc'>
+            <Text
+              className='InfoCardLabel'
+              weight={400}
+              tint={50}
+              size={12}
+              color='primary'
+            >
+              Author
+            </Text>
+            <div className='InfoCardValue flex fac'>
+              <Icon name='avatar' fontSize={14} />
+              <Text
+                className='InfoCardValueText'
+                weight={500}
+                size={12}
+                color='primary'
+              >
+                {props.data.commit?.author}
+              </Text>
+            </div>
+          </div>
+          <div className='InfoCard flex fdc'>
+            <Text
+              className='InfoCardLabel'
+              weight={400}
+              tint={50}
+              size={12}
+              color='primary'
+            >
+              Hash
+            </Text>
+            <div className='InfoCardValue flex fac'>
+              <Icon name='hash' fontSize={14} />
+              <Text
+                className='InfoCardValueText'
+                weight={500}
+                size={12}
+                color='primary'
+              >
+                {props.data.commit?.hash}
+              </Text>
+            </div>
+          </div>
+          <div className='InfoCard flex fdc'>
+            <Text
+              className='InfoCardLabel'
+              weight={400}
+              tint={50}
+              size={12}
+              color='primary'
+            >
+              Timestamp
+            </Text>
+            <div className='InfoCardValue flex fac'>
+              <Icon name='time' fontSize={14} />
+              <Text
+                className='InfoCardValueText'
+                weight={500}
+                size={12}
+                color='primary'
+              >
+                {`${moment(props.data.commit?.timestamp).format(
+                  'DD MMMM YYYY HH:MM A',
```

Comment:
Please add date format to constants and use here constant value. 

src/config/dates/dates.ts